ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -157,27 +150,72 @@ export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
                     closeOnClick
                     content={
                       <React.Fragment>
-                        <MenuItem style={styles.presetMenuItem}>General purpose analysis</MenuItem>
-                        <MenuItem style={styles.presetMenuItem}>Genomics analysis</MenuItem>
+                        {
+                          fp.filter(['displayName', 'General Analysis'], runtimePresets)
+                            // regular .map used here bc fp map won't provide index as an iteratee argument
```

Comment:
I'm cool with using regular map but the lodash docs says that they do provide an index? https://lodash.com/docs/4.17.15#map

Reply:
We're not using lodash per se, we're using lodash/fp, which does not provide an index.